packages/web-pixels-extension/src/types/PixelEvents/index.ts

```diff
@@ -338,12 +346,6 @@ export interface PixelEvents {
    */
   collection_viewed: PixelEventsCollectionViewed;
 
-  /**
-   * This event represents any custom events emitted by partners or merchants
-   * via the `publish` method
-   */
-  custom_event: PixelEventsCustomEvent;
```

Comment:
Removed this key from `PixelEvents` since it's not a standard event. `PixelEventsCustomEvent` would now be `CustomEvent`